dlc-manager/src/manager.rs

```diff
@@ -466,7 +489,7 @@ where
         sign_message: SignDlc,
         e: Error,
     ) -> Result<R, Error> {
-        error!("Error in on_sign {}", e);
+        error!("Error in on_sign {:?}", e);
```

Comment:
Why this?

Reply:
this was helping me debug, can remove if you don't want it

Reply:
Maybe better to improve the error printing with what you were missing?

Reply:
You haven't reverted this AFAICT